example/integration_test/webrtc_test.dart

```diff
@@ -299,16 +299,13 @@ void main() {
       'audio/opus',
       'audio/red',
       'audio/G722',
-//       'audio/ILBC',
```

Comment:
`audio/ILBC` was removed from the recent version of libwebrtc due to its declining usage:

https://issuetracker.google.com/issues/372395680